### DISCUSSION TITLE
Touchlink Reset Device

### DISCUSSION THREAD

**Discussion by 8art-5imson** [Koenkk/zigbee2mqtt#8114] [Devices]
Trying now for several days to reset Hue Lamps after zigbee failed. Re-Paired all devices except Hue Lamps, as i discovered that touchlink reset no longer works with my zzh co-ordinator. So I put an old sd card with node red and Zigbee2MQTT version 1.18.2 on it that i was certain i used to use to reset the lamps by sending an empty payload to zigbee2mqtt/bridge/request/touchlink/factory_reset. Only to discover that this no longer works. So I then tried it with a CC2531 adapter only to discover that this dosn't work any more either. Does anyone know at which version the touchlink stopped working ? as i would like to build a battery powered device with an old pi and a CC2531 stick to be able to go around resetting all my hue Lamps (about 20 GU10s). Also is there any sign of the CC2652R chip ever working with TouchLink?

**Comment by Loic691** [comment]
CC2531 seams works. So you 'd like to open an issue
